coldpress.py

```diff
@@ -57,26 +57,32 @@ def __init__(self):
         self.jobs = []
         self.log = []
         self.resources = {"nodes": {}}
-        ret = self.create_job() 
-        nodes = oc.selector('nodes').objects()
+        ret = self.create_job()
+        # Initialize Kubernetes client
+        try:
+            config.load_incluster_config()
+        except:
+            config.load_kube_config()
```

Comment:
This is probably something for a future PR, but I would move the kubernetes initialization out of this class. There are no tests now, but at some point (e.g., yesterday) we're going to want tests, and the ability to test things out without implicitly initializing the kubernetes connection (e.g., because we're providing a fake client for testing) will be valuable.